pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -338,6 +338,7 @@ spec:
       dnsConfig:
         nameservers:
         - "8.8.8.8"
+        - "2001:4860:4860::8888"
```

Comment:
this is probably a bad idea. We _cannot_ reach this. So it may hang.

Having just ipv4 gives :`Error 1/1 requests had errors; first error: Get "http://asd": dial tcp: lookup asd on 8.8.8.8:53: dial udp 8.8.8.8:53: connect: network is unreachable` but it does so _immediately_